CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -1475,6 +1475,7 @@ protected function postProcessMembership(
         $isTest,
         $isRecurForFirstTransaction
       );
+      $paymentResults[] = array('contribution_id' => $paymentResult['contribution']->id, 'result' => $paymentResult);
```

Comment:
This can produce a notice error for `Undefined 'contribution' in $paymentResult`(did once for me). I see we already have a `!empty($paymentResult['contribution'])` check just below this line. Can we move this inside the if check just to avoid unwanted e-notices ?